src/Notes/WC_Admin_Note.php

```diff
@@ -504,6 +537,36 @@ public function clear_actions() {
 		$this->set_prop( 'actions', array() );
 	}
 
+	/**
+	 * Set note layout.
+	 *
+	 * @param string $layout Note layout.
+	 */
+	public function set_layout( $layout ) {
+		$valid_layouts = array( 'banner', 'plain', 'thumbnail' );
+		if ( in_array( $layout, $valid_layouts ) ) {
```

Comment:
Should this throw an error so that an invalid layout doesn't get swallowed?

Reply:
Sounds good to me 🙂
I added it in the commit [50efe8e](https://github.com/woocommerce/woocommerce-admin/pull/4262/commits/50efe8e3080f604b386172b4354e7b1f72573af9)